sigs.yaml

```diff
@@ -3439,15 +3439,7 @@ workinggroups:
     - github: serathius
       name: Marek Siarkowicz
       company: Google
-  meetings:
-  - description: Regular Meeting
-    day: Thursday
-    time: "15:30"
-    tz: London-UK
-    frequency: biweekly
-    url: https://zoom.us/j/96716142646?pwd=VmgrN29sbmhDREp3R0NtZlpGSlZ4Zz09
-    archive_url: https://docs.google.com/document/d/1R9bZ34L9vR1ftH0dFeOp-j50lLh5ijKVwwXJ3LDrY4I/edit?usp=sharing
-    recordings_url: https://www.youtube.com/playlist?list=PL69nYSiGNLP2GY3so1z2Cnkvkt5A_x1Aw
+  meetings: []
```

Comment:
Should we keep the links to notes and recordings?

Reply:
Let's check... no, the tool doesn't like a `meetings` entry without time.

There is a `BEGIN CUSTOM CONTENT` section in README.md, but adding anything there leads to a verification failure. :shrug:

Reply:
Ok, lets assume people can do git history.